### ISSUE TITLE
ajoberstar.github.io/bintray-backup/ redirects to insecure URL

### ISSUE THREAD

**Issue by joschi** [ajoberstar/grgit#335]
The URL of the Bintray backup mentioned in the README of this project is redirecting to another location with an insecure protocol (HTTP) instead of keeping the connection secure (HTTPS):
```
# curl -IL https://ajoberstar.github.io/bintray-backup/
HTTP/2 301
server: GitHub.com
location: http://ajoberstar.org/bintray-backup/

HTTP/1.1 301 Moved Permanently
Server: GitHub.com
Content-Type: text/html
Location: https://ajoberstar.org/bintray-backup/

HTTP/2 200
server: GitHub.com
content-type: text/html; charset=utf-8
```

Unfortunately this is causing issues with Gradle 7.0 and later because insecure repositories (served via HTTP) aren't permitted anymore by default.

Example: https://travis-ci.org/github/n0mer/gradle-git-properties/jobs/768161218#L243-L250
```
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not resolve org.ajoberstar.grgit:grgit-core:4.1.0.
     Required by:
         project :
      > Could not resolve org.ajoberstar.grgit:grgit-core:4.1.0.
         > Could not get resource 'https://ajoberstar.github.io/bintray-backup/org/ajoberstar/grgit/grgit-core/4.1.0/grgit-core-4.1.0.pom'.
            > Redirecting from secure protocol to insecure protocol, without explict opt-in, is unsupported. 'https://ajoberstar.github.io/bintray-backup/' is redirecting to 'http://ajoberstar.org/bintray-backup/org/ajoberstar/grgit/grgit-core/4.1.0/grgit-core-4.1.0.pom'. Switch Maven repository 'ajoberstar-backup(https://ajoberstar.github.io/bintray-backup/)' to redirect to a secure protocol (like HTTPS) or allow insecure protocols. See https://docs.gradle.org/7.0/dsl/org.gradle.api.artifacts.repositories.UrlArtifactRepository.html#org.gradle.api.artifacts.repositories.UrlArtifactRepository:allowInsecureProtocol for more details.
```

**Comment by joschi** [comment]
Maybe enforcing HTTPS for GitHub Pages for the respective repository would help?
https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https#enforcing-https-for-your-github-pages-site

**Comment by ajoberstar** [comment]
Try `https://ajoberstar.org/bintray-backup/`. I put the wrong URL in the README (updated now).

**Comment by joschi** [comment]
Using the final URL of the redirections is working as expected.

Thanks for changing the URL in the README file.

Fixed in ccbe02b2288817d145cbe89a4ef18e454965ad4f.
